lib/moab/verification_result.rb

```diff
@@ -15,34 +16,30 @@ class VerificationResult
     attr_accessor :subentities
 
     # @param entity [Object] The name of the entity being verified
-    def initialize(entity)
+    # @param verified [Boolean]
+    # @param details [Hash]
+    def initialize(entity, verified = false, details = nil)
       @entity = entity
-      @verified = false
-      @details = nil
+      @verified = verified
+      @details = details
       @subentities = Array.new
     end
 
-    # @param entity [Symbol] The name of the entity being verified
+    # @param entity [#to_s] The name of the entity being verified
```

Comment:
copy pasta fail?  it is a Symbol, not a "#to_s"

Reply:
No, that is deliberate duck-typing.  It is literally anything that responds to `#to_s`, since that is the only thing we do with it.  In no respects to we actually rely on it being a Symbol (though that is expected).